### ISSUE TITLE
What is the type of loss function in the pretrained Conv-TasNet model?

### ISSUE THREAD

**Issue by Tigerwww** [asteroid-team/asteroid#523]
Hi, I need your help.  I'm using your pre-trained Conv-TasNet: mpariente/ConvTasNet_Libri1Mix_enhsingle_8k. But I'm wondering if you still remembered which kind of loss function was used.  SI-SNR or SD-SDR?  Looking forward to your reply.

**Comment by mpariente** [comment]
SI-SNR

**Comment by Tigerwww** [comment]
Thank you very much for answering!
